trakt/calendar.py

```diff
@@ -60,14 +66,23 @@ def _get(self):
     def _build(self, data):
         """Build the calendar"""
         self._calendar = []
-        for episode in data:
-            show = episode.get('show', {}).get('title')
-            season = episode.get('episode', {}).get('season')
-            ep = episode.get('episode', {}).get('number')
-            e_data = {'airs_at': airs_date(episode.get('first_aired')),
-                      'ids': episode.get('episode').get('ids'),
-                      'title': episode.get('episode', {}).get('title')}
-            self._calendar.append(TVEpisode(show, season, ep, **e_data))
+        for cal_item in data:
+            show_data = cal_item.get('show', {})
+            episode = cal_item.get('episode', {})
+            first_aired = cal_item.get('first_aired')
+            season = episode.get('season')
+            ep_num = episode.get('number')
+            extract_ids(show_data)
+            show_data.update(show_data)
+            e_data = {
+                'airs_at': airs_date(first_aired),
+                'ids': episode.get('ids'),
+                'title': show_data.get('title'),
+                'show_data': TVShow(**show_data)
```

Comment:
Tagged the wrong line. I put the show object in show_data. But mostly in the app show property has the show title.

Reply:
I think this line should be fine